tutorials/joint_1d_3d_analysis.ipynb

```diff
@@ -4,22 +4,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Joint Crab analysis with 1D and 3D datasets\n",
-    "\n",
-    "**TODO: move references on how to create the datasets at the beginning. Change title and scope to discuss joint modeling and fitting of the Crab nebula.\n",
-    "Discuss at the beginning the 2 types of datasets to be used. Explain better\n",
-    "what the Fermi model is.**"
+    "# Joint modeling, fitting, and serialization\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This tutorial illustrates how to run a joint analysis with different datasets.\n",
+    "This tutorial illustrates how to perfom a joint modeling and fitting of the Crab Nebula spectrum using different datasets.\n",
     "We look at the gamma-ray emission from the Crab nebula between 10 GeV and 100 TeV.\n",
-    "The spectral parameters are optimized by combining a 3D analysis of Fermi-LAT data, a ON/OFF spectral analysis of HESS data, and flux points from HAWC.  \n",
+    "The spectral parameters are optimized by combining a 3D analysis of Fermi-LAT data, a ON/OFF spectral analysis of HESS data, and flux points from HAWC.\n",
+    "\n",
+    "In this tutorial we are going to use pre-made datasets. We prepared maps of the Crab region as seen by Fermi-LAT using the same event selection than the [3FHL catalog](https://arxiv.org/abs/1702.00664) (7 years of data with energy from 10 GeV to 2 TeV). For the HESS ON/OFF analysis we used two observations from the [first public data release](https://arxiv.org/abs/1810.04516) with a significant signal from energy of about 600 GeV to 10 TeV. These observations have an offset of 0.5° and a zenith angle of 45-48°. The HAWC flux points data are taken from a [recent analysis](https://arxiv.org/pdf/1905.12518.pdf) based on 2.5 years of data with energy between 300 Gev and 300 TeV. \n",
+    "\n",
+    "More details on how to prepare datasets with the high and low level interfaces are available in these tutorials: \n",
     "\n",
-    "\n"
+    "- https://docs.gammapy.org/0.14/notebooks/fermi_lat.html\n",
```

Comment:
@QRemy - Please fix the links to other tutorials to use the relative cross-linking as desribed here: https://docs.gammapy.org/dev/development/howto.html#dealing-with-links-and-notebooks

Currently you link once to `v0.14` and once to `dev` as raw URLs.